migrateUpWithWrapper.mjs

```diff
@@ -0,0 +1,36 @@
+/**
+ * Hacky script that wraps the 'node-pg-migrate' program
+ * and make sure that logged errors do not leak some
+ * sensitive information such as db password for instance..
+ * */
+import dotenv from 'dotenv';
+import { spawn } from 'child_process';
+
+dotenv.config();
+
+const dbUser = process.env.PGUSER;
```

Comment:
I you want that script to work, make sure you are using env vars with the `PG` prefix vs `DATABASE`.